bindings/wasm/bindings.js

```diff
@@ -291,74 +487,85 @@ Module.setup = function() {
     return manifold;
   };
 
-  Module.Manifold.prototype = Object.create(ManifoldCtor.prototype);
+  Module.Manifold.ofMesh = function(mesh) {
+    return new Module.Manifold(mesh);
+  };
 
-  Module.cube = function(...args) {
+  Module.Manifold.tetrahedron = function() {
+    return Module._Tetrahedron();
```

Comment:
Necessary?

Reply:
Yes, the class_function method of embind was not working for me for setting up the static methods (compiles, but then the function is not actually available in examples.js (function undefined). Maybe because of the way that the object is being replaced in binding.js? I imagine there is some reorganization that would not require this, but the way we have it setup right now it doesn't work.

Reply:
Interesting, thanks.